hardening.mk.inc

```diff
@@ -1,5 +1,4 @@
 CFORTIFY=-O2 -D_FORTIFY_SOURCE=2
-CASLR=-fPIC -fPIE
```

Comment:
This wasn't even used and didn't realy make sense at all, mixing PIC with PIE.

Reply:
:+1: